packages/manager/modules/sharepoint/package.json

```diff
@@ -20,7 +20,9 @@
   "peerDependencies": {
     "@ovh-ux/ng-ovh-utils": "^14.0.3",
     "@ovh-ux/web-universe-components": "^7.0.0",
+    "@uirouter/angularjs": "^1.0.15",
```

Comment:
Latest is `^1.0.22` if I'm correct, any reason that would prevent us from using it instead ?

Reply:
No, use the same version as other modules.